index.js

```diff
@@ -1,310 +1,333 @@
-const express = require('express')
-const https = require('https')
-const fs = require('fs');
-const cors = require('cors')
-const path = require('path');
-const app = express()
-const port = 8000
+const express = require("express");
+const https = require("https");
+const fs = require("fs");
+const cors = require("cors");
+const path = require("path");
+const app = express();
+const port = 8000;
+const crypto = require("crypto");
+const dotenv = require("dotenv");
 
-app.use(cors());
-app.use(express.json());
+// Enable environment variables
+dotenv.config({
+  path: "./.env",
+});
 
-// Front end endpoints
-app.get('/', (req, res) => {
-  res.send('Hello World!')
-})
+// Parse JSON bodies
+app.use(express.json());
 
-app.get('/auth', (req, res) => {
-  console.log("Auth Happened!")
-  res.sendFile(path.join(__dirname, '/auth.html'))
-})
+// Enable CORS (https://developers.asana.com/docs/security)
+app.use(cors());
```

Comment:
consider: might be relatively lightweight to add some specific CORS settings here, but it's not strictly necessary.